slim/nets/resnet_v2.py

```diff
@@ -115,7 +117,7 @@ def resnet_v2(inputs,
               global_pool=True,
               output_stride=None,
               include_root_block=True,
-              spatial_squeeze=True,
+              spatial_squeeze=False,
```

Comment:
Were you able to get this model running without it? From what I remember, I needed `spatial_squeeze=True` in order to get it to run properly. (To be fair though, that was multiple months ago.)